internal/ci/ethereum_test

```diff
@@ -25,45 +34,52 @@ cd examples/echo_server
 yarn install
 truffle migrate
 node echo.js &
+sleep 1
+
+########################
+## runlog
+########################
+
+./send_runlog_transaction.js
+sleep 2
+
+# Check echo count
+count=`curl -sS localhost:6690/count`
+assert "Echo count" 1 $count
+
+# Check CL counts
+cd ../../
+
+## Check job counts
+jobs=`cldev -j j | jq length`
```

Comment:
Should we document the `jq` requirement somewhere?

Reply:
I'm indifferent. I do think this script assumes a lot of things are installed so how do we draw the line between `jq` and other dependencies (yarn, curl, node)? It's preinstalled on the CI servers because it's prevalent in the ops world. Are you just saying something like?

```bash
## jq is used to parse json: https://stedolan.github.io/jq/
```